### DISCUSSION TITLE
Need Help for Teacher affiliation verification

### DISCUSSION THREAD

**Discussion by khammami** [community/community#47055] [GitHub Education]
Hello,

I'm used to verify my status in the past years with a scan of my contract which is in Arabic, but this time it doesn't accept it! Should add a translation? 

The university doesn't provide a translated version, should I translated it by a professional?

**Comment by jge162** [comment]
Yes, an Admin posted the other day to please send all requests in english.

**Reply by khammami** [reply]
Thanks for your reply, I just want to know if they accept a translated copy by a professional because unfortunately the university doesn't provide it in English!

**Reply by samus-aran** [reply]
Hi @khammami 👋 

Yes, we do accept translated evidence. Write the evidence in English and take a photo of both the original and translated evidence side by side.

Hope this helps. Let us know how you get on 👍

**Reply by khammami** [reply]
Thanks @samus-aran I'll get it translated then try again.
